### DISCUSSION TITLE
Adding custom REPL commands

### DISCUSSION THREAD

**Discussion by khill-fbmc** [hongaar/bandersnatch#403] [Ideas]
Just throwing [this at the wall](https://nodejs.org/dist/latest-v18.x/docs/api/repl.html#replserverdefinecommandkeyword-cmd), is it possible already with your lib?

**Comment by hongaar** [comment]
The Repl server is only used to pass commands to yargs. The commands feature of the Node Repl server is not used. Can you explain in more detail what you want to achieve?

**Reply by kevinkhill** [reply]
☝🏻.... I don't know, just thought it was a cool feature? Haha, I'm not sure.

I'm guilty of future feature building "it would be cool if..."
